codecov.yml

```diff
@@ -1 +1,10 @@
+flag_management:
+  default_rules: # the rules that will be followed for any flag added, generally
+    carryforward: true
+    statuses:
+      - type: project
+        target: auto
+        threshold: 1%
+      - type: patch
+        target: 90%
 comment: off
```

Comment:
do we need this or can we just take advantage of org settings?

Reply:
we need at least the `flag_management` thing, let me try without that...

Reply:
null value not allowed there but I can ditch all teh statuses: bits